src/features/deposit/components/Form/DepositFormRouter/index.tsx

```diff
@@ -180,3 +166,40 @@ export const DepositFormRouter = ({ onSubmit }: DepositFormRouterProps) => {
     </Form>
   )
 }
+
+function filterGroupedTokens(
+  token: SwappableToken,
+  blockchain: BlockchainEnum
+) {
+  if (!blockchain) return true
```

Comment:
`blockchain` cannot be falsy

Reply:
Yes, in groupedTokens asset

Reply:
`blockchain: BlockchainEnum` means it contains value of BlockchainEnum and BlockchainEnum does not contain any falsy values. I suppose something is wrong here.